appstudio-controller/controllers/appstudio.redhat.com/snapshotenvironmentbinding_controller_test.go

```diff
@@ -758,6 +758,81 @@ var _ = Describe("SnapshotEnvironmentBinding Reconciler Tests", func() {
 			Expect(binding.Status.ComponentDeploymentConditions[0].Message).To(Equal("1 of 2 components deployed"))
 		})
 
+		// Similar to previous test except to test the SnapshotEnvironmentBinding's statuses of its GitOpsDeployments
+		It("sets the status fields of the GitOpsDeployments of a SnapshotEnvironmentBinding", func() {
+			By("Creating SnapshotEnvironmentBinding CR in cluster.")
+			err := bindingReconciler.Create(ctx, binding)
+			Expect(err).To(BeNil())
+
+			By("Checking status field before calling Reconciler")
+			binding = &appstudiosharedv1.SnapshotEnvironmentBinding{}
+			err = bindingReconciler.Client.Get(ctx, request.NamespacedName, binding)
```

Comment:
Nit, it can also be `bindingReconciler.Get`

Reply:
Done.